### PR TITLE
Add meeting minutes from April 16 and 23

### PR DESCRIPTION
**Pull request by ptomato**
Tagging everyone with review access who was in these meetings

### CODE REVIEW

File: meetings/agenda-minutes-2020-04-16.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- PFC: One of the use-cases for custom time-zones is that if you wanted to do date-time arithmetic before time-zones were standardized.
+- DE: In case of out-of-date implementations, we could redirect the user to the polyfill.
+- SFC: The approach of nuking the existing builtins completely feels awkward to me. You want the ability to patch single calendars or time-zones without getting rid of all the rest.
+- DE: The problem is, there are alternative solutions for all of these things.
+- SFC: That’s why I proposed two things, they’re on either side of the spectrum.
+- DE: I think there’s a cost of all this indirection. My point is, we should have a smaller surface area. The problem is, the stuff around custom calendars and timezones keeps getting more and more complicated and I don’t feel that this is good enough justification.
+- SFC: I now see DE’s point, and I see the connection to Intl. A lot of people built wrappers on top of Intl to deal with the fact that Intl doesn’t have the ability to add data. The difference here is that Temporal is sort of a fundamental data type. Maybe we could move Timezones and Calendars out of Temporal and into a more data-driven namespace.
```
Comment (kaizhu256) [inline meetings/agenda-minutes-2020-04-16.md:70]: > The difference here is that Temporal is sort of a fundamental data type

i disagree "temporal" is treated as fundamental datatype in practice.  subconsciously or not, product-developers usually implement "temporal" as collection of static-functions to achieve some ux-objective (through manipulation and message-passing of serialized isostrings between ui and datastore).

nobody cares about a Date/DateTime-object in and of itself.  the only thing that matters is the isostring-representation  to be integrated with dom-elements, ingested into sql-database, or message-passed between clients and servers.

Comment (ljharb) [reply meetings/agenda-minutes-2020-04-16.md:70]: No, they do not usually do that; they usually reach for a library like moment.js which is treated as a fundamental data type.

You are incorrect that the string representation is the only thing that matters; you've asserted this dozens of times and I've attempted to correct your assertion every time, so far with no success.

Comment (kaizhu256) [reply meetings/agenda-minutes-2020-04-16.md:70]: they use moment.js to again, ultimately create static-functions that inputs an isotring, and outputs an isostring (for some ux-specific objective).

**Comment by ryzokuken** [comment]
Merging this! Thanks again @ptomato.

**Pull request merged**